README.md

```diff
@@ -592,6 +598,10 @@ const options: FeedClientConfiguration = {
 client.feed(fql`Product.all().eventSource()`, options);
 ```
 
+You can reuse
```

Comment:
@erickpintor Can you confirm this? Also let me know if there are any potential pitfalls to point out here. Thanks!

Reply:
That's true. Cursors are meant to be track your position wrt. the event feed you're consuming. The event source shape is deterministic, though. You can either use the same event source you already have or regenerate it via the same FQL query.

Reply:
Awesome. Thank you!